Gruntfile.js

```diff
@@ -165,6 +165,21 @@ module.exports = function (grunt) {
                 src: 'src/main/webapp/js/min/script.js',
                 dest: 'src/main/webapp/js/min/script.clean.js'
             }
+        },
+        protractor : {
+            options : {
+                keepAlive : true,
+                noColor : false,
```

Comment:
In pom file we use <arguments>--no-color</arguments> in 'frontend-maven-plugin' configuration